src/context/AppProvider/AppContext.tsx

```diff
@@ -138,22 +139,22 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
         for (const chain of supportedChains) {
           const adapter = chainAdapterManager.byChain(chain)
+          const { chainNamespace, chainReference } = chainTypeToMainnetChainParts(chain)
+          const chainId = toChainId({ chainNamespace, chainReference })
 
           switch (chain) {
```

Comment:
Also not directly related, but do we just want to update the switch to use `chainId` over `ChainTypes`? Totally reasonable to just wait for `ChainTypes` removal too, just a thought.

Reply:
Addressed in https://github.com/shapeshift/web/pull/1892/commits/effc6bd85de59333b65a003f04274994f77b8b5d